React/Views/RCTScrollView.m

```diff
@@ -317,8 +317,8 @@ - (void)setFrame:(CGRect)frame
   CGSize boundsSize = self.bounds.size;
 
   self.contentOffset = CGPointMake(
-    MAX(0, MIN(originalOffset.x, fullContentSize.width - boundsSize.width)),
-    MAX(0, MIN(originalOffset.y, fullContentSize.height - boundsSize.height)));
+    MAX(-contentInset.left, MIN(originalOffset.x, fullContentSize.width - boundsSize.width)),
```

Comment:
And for context, that diff is supposed to fix the problem known as https://github.com/facebook/react-native/issues/13566